src/app.ts

```diff
@@ -41,12 +43,18 @@ const main = async () => {
     credential: admin.credential.cert(process.env.FCM_AUTH_KEY_PATH),
   });
 
-  app.post('/collect/', (req, res) => {
+  app.post('/collect/articles/', (req, res) => {
```

Comment:
has this change been communicated with frontend? changing this will break current apps fetching articles

Reply:
I'm not sure if frontend uses these endpoints